servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -314,7 +314,7 @@ def query_jobs(body, **kwargs):
 
     response = requests.post(
         _get_base_url() + '/query',
-        json=cromwell_query_params(query, page, query_page_size),
+        json=cromwell_query_params(query, page, query_page_size, headers),
```

Comment:
It looks like we're sending `headers` both as headers and in the POST JSON - is this intended?

Reply:
the JSON uses `headers` as a parameter, not within the result